thinkhazard/static/less/report.less

```diff
@@ -149,6 +149,26 @@ body {
   top: 10px;
   left: 10px;
   z-index: 2;
+
+  height: 42px;
+  margin: 5px;
+  background: rgba(255,255,255, 0.5);
+  border-radius: 5px;
+  border: 2px solid rgba(255,255,255,0.75);
+  padding: 5px;
```

Comment:
Let's keep the default value for padding in buttons.
Or, let's change it globally for all the xs buttons (.btn-xs).